editor/layouts/layout-tools.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Tools'
+sidebarTitle: 'Tools'
+description: 'There are several Layout tools available in Rive to build your responsive UI or content'
+---
+
+import { YouTube } from '/snippets/youtube.mdx'
+
+## Getting Started with Layouts
+
+There are a number of ways to start adding layouts to your designs.
+
+- Layout Tools in the Arrangement Tools menu
+- Wrap in Layout
+- Add Child Layout
+- Dragging and Dropping Layouts
+
+---
+
+### Layout Tools in the Arrangement Tools Menu
+
+A selection of layout options are available in the Arrangement Tools menu. These tools allow you to draw a Layout in 2 ways:
+
+1. When dragged directly onto the Artboard, it will create an Absolute Layout positioned relative to the Top Left of the Artboard.
+2. When dragged on top of another Layout, it will create a Relative child Layout inside the Layout it was drawn on top of.
+
+![Image](https://ucarecdn.com/53f42ad0-4045-41c4-bcfe-90a40ad0dc1d/)
```

Comment:
Any particular reason for uploading this to this CDN instead of adding it to the repository? cc @dskuza, have we looked into ucarecdn?